network-config.copy

```diff
@@ -0,0 +1,318 @@
+live:
+  - name: Ethereum
```

Comment:
This appears to be a copy of brownie's current version of "network-config.yaml".

Which means it won't evolve with future versions of "network-config.yaml".

There's gotta be a more lightweight way.

Remember: when brownie gets installed, it should also include  "network-config.yaml".

This PR should be a "light touch" that lightly edits the file, automatically, using e.g. ">", "|" and "sed".